src/qt/walletcontroller.h

```diff
@@ -72,19 +73,28 @@ class OpenWalletActivity : public QObject
     Q_OBJECT
 
 public:
-    OpenWalletActivity(WalletController* wallet_controller, const std::string& name);
+    OpenWalletActivity(QWidget* parent = nullptr);
```

Comment:
Should be `explicit` :-)  Please help review #14505 to get automatic `explicit` checking for this - no need for human intervention here :-)